fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/MessageValidator.kt

```diff
@@ -173,6 +175,19 @@ object MessageValidator {
       return this
     }
 
+    /**
+     * Helper method to check that a write-in is valid.
+     *
+     * @param input the encrypted write-in to check
+     * @param field name of the field (to print in case of error)
+     * @throws IllegalArgumentException if the string is empty or not a URL-safe base64 encoding
+     */
+    fun isValidWriteIn(input: String?, field: String): MessageValidatorBuilder {
+      requireNotNull(input) { "Write-in is enabled but encrypted write-in is null" }
+      isNotEmptyBase64(input, field)
+      return this
+    }
+
```

Comment:
I think we should try to keep the MessageVerify as generic as possible, and I find this function a bit redundant and very specific to just a single use. I'm assuming that the purpose was to have a more custom message in case of null input, but if that's the case it could be done differently (like extend the isNotEmptyBase64 to support custom messages).

The `isNotEmptyBase64` function already performs this check for you (not null), so in the end this function is just a wrapper with no strong need for it.